src/utils/package.ts

```diff
@@ -0,0 +1,50 @@
+export type PackageDotJson = {
```

Comment:
l: Do we wanna rename this file to something like `utils/package-json.ts`? `package.ts` sounds a bit ambiguous to me.